generation/WinSDK/Partitions/Dxgi/settings.rsp

```diff
@@ -13,3 +13,11 @@ _D3DCOLORVALUE
 <IncludeRoot>/um/dxprogrammablecapture.h
 --namespace
 Windows.Win32.Graphics.Dxgi
+--with-attribute
+DXGI_MULTIPLANE_OVERLAY_YCbCr_FLAGS=Flags
+DXGI_OFFER_RESOURCE_FLAGS=Flags
+DXGI_OUTDUPL_FLAG=Flags
```

Comment:
This flag has one `DXGI_OUTDUPL_COMPOSITED_UI_CAPTURE_ONLY = 1` constant, but I couldn't find in what API it is supposed to be used. Only the constructor for output duplication has as `Flags` field that "should always be zero":


https://learn.microsoft.com/en-us/windows/win32/api/dxgi1_5/nf-dxgi1_5-idxgioutput5-duplicateoutput1

Reply:
Looks like a question for the DX repo. Suspect this is was meant to stay internal. But a real world example from OS components verifies your claim: `pObject->DuplicateOutput1(device, DXGI_OUTDUPL_COMPOSITED_UI_CAPTURE_ONLY, ...)`

Reply:
Sounds good, in that case the documentation should be updated too.

What's the desired way to assign the enum, emitter.settings.rsp or listing it in enums.json?

Reply:
Think we prefer `enums.json` these days!

Reply:
@riverar that might mean we could move more of the `--with-attribute` here - with explicit reassignments in `emitter.settings.rsp` - and consolidate them into `enums.json`?

Looks like there's also `--enumMakeFlags` in `emitter.settings.rsp` doing a similar thing 😅

Reply:
> But a real world example from OS components verifies your claim: `pObject->DuplicateOutput1(device, DXGI_OUTDUPL_COMPOSITED_UI_CAPTURE_ONLY, ...)`

@riverar where'd you find this example?

Reply:
`bcastdvruserservice.dll!Windows::Media::Capture::Internal::DDACapture::AcquireOutputDuplication` (26217.5000), reverse engineered 😅

Reply:
Posted the question at https://github.com/microsoft/DirectX-Headers/issues/128, let's see.